src/assets/scss/utils/mixins.scss

```diff
@@ -1,27 +0,0 @@
-// px to rem
-@function rem($pixels, $context: $mainFontSize) {
```

Comment:
I can't find where you use this function (as well as `@mixin placeholder`). Do not forget to remove unnecessary code